src/styles/Blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+$day-size: 100px;
+$calendar_gap: 1px;
+$calendar_padding: 10px;
+$days:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: (7 * $day-size) 
+    + (6 *$calendar_gap) 
+    + (2 * $calendar_padding);
+
+  gap: $calendar_gap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $day-size;
+    height: $day-size;
+
+    background-color: #eee;
+    border: 1px solid #000;
+
+    transition: transform 0.5s ease;
```

Comment:
i think it would be good to add a color transition

![image](https://user-images.githubusercontent.com/113522334/201029092-064dd528-2eba-4684-a9bb-55d0faa6ddef.png)

Reply:
fixed